src/projects/aws-eks/index.ts

```diff
@@ -1063,3 +1063,68 @@ const jh = new k8s.helm.v3.Release(
     provider: cluster.provider
   }
 );
+
+// Install fluent-bit
+const fluentbitOutputConfig = `
+[OUTPUT]
+    Name es
+    Match kube.*
+    Host elasticsearch-master
+    # Trace_Output On
+    Trace_Error On
+    Logstash_Format On
+    Retry_Limit False
+    Replace_Dots On
+    
+[OUTPUT]
+    Name es
+    Match host.*
+    Host elasticsearch-master
+    # Trace_Output On
+    Trace_Error On
+    Logstash_Format On
+    Logstash_Prefix node
+    Retry_Limit False
+    Replace_Dots On
+`;
+
+const fluentbit = new k8s.helm.v3.Release(
+  'fluent-bit', {
+    chart: 'fluent-bit',
+    name: 'fluent-bit',
+    namespace: aiMonitorNs.metadata.name,
```

Comment:
@phamhoangtuan you mentioned its not being installed in `aitomatic-monitor namesapce`. Is this value not being set correctly or is there a typo?

Reply:
I tried both `aiMonitorNs.metadata.name` and `aiMonitorNs.id` but not successfully. Using `Chart` then it can be installed in the correct ns